packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -431,11 +553,11 @@ function validateId(
     if (!id) {
       id = "Function name is invalid";
```

Comment:
Can id be falsy here for an enum as well? This only says "Function name" is invalid.

Reply:
I changed to use `validateName` instead of `validateId` for a full validation.